Kirill_Samsonov/2/lib/spec/domain_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require_relative '../domain'
+
+describe Domain do
+  subject { Domain.new(name) }
+
+  describe 'first level' do
```

Comment:
To follow the best practices you should change these tests in the following way:
- put `subdomain?` method tests to one block
- use `when` in `context`/`describe` block name and use a more understandable name
- use a verb in the `it` block name
- do not compare with `true`/`false` if you don't need exactly these values - often it is important for value to be truthy/falsy, not exactly `true`/`false`. So in your case it is up to you.

```suggestion
  describe '#subdomain?' do
    context 'when url has no subdomains' do
       let(:name) { 'domain.com' }
       
       it 'returns falsey value' do
         expect(subject.subdomain?).to be_falsey
       end
     end
     
     context 'when url has subdomains' do
       let(:name) { 'sub.domain.com' }
       
       it 'returns truthy value' do
         expect(subject.subdomain?).to be_truthy
       end
     end
   end
```